eduvpn/main.py

```diff
@@ -13,20 +13,16 @@
 from argparse import ArgumentParser
 from eduvpn import __version__
 from eduvpn import config
+from typing import Tuple
+from eduvpn.ui import EduVpnApp
 
 
 logger = logging.getLogger(__name__)
 log_format = format_ = '%(asctime)s - %(levelname)s - %(name)s - %(message)s'
 
 
-def parse_args():
-    """
-    Parses command line arguments:
-
-    returns:
-        (logging level (int), secure internet uri (str), institute access uri (str), verify key (str))
-
-    """
+def parse_args():  # type: () -> Tuple[int, str, str, str, bool]
+    """Parses command line arguments."""
```

Comment:
hier ook verdwenen arg doc. Het is wel handig om te weten wat er in de tuple zit.